### DISCUSSION TITLE
Add google fonts in Mantine

### DISCUSSION THREAD

**Discussion by I-am-tk** [mantinedev/mantine#1160] [Q&A] [answered]
I pasted the `link` tag in the `index.html` file. And then passed the 
```
<MantineProvider theme={{ fontFamily: ["Inter", "sans-serif"] }}>
      Your app here
    </MantineProvider>
```
But the font family is not Inter. Do I have to set `font-family` for each Mantine component? Please see https://codesandbox.io/s/damp-mountain-85l5dm?file=/src/App.tsx

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/s/keen-nobel-6qbci2?file=/src/App.tsx

**Reply by I-am-tk** [reply]
Thanks,
This works only for `Mantine Components`. For it to work on other components, I can add the `fontFamily` property in the body element using the `Global` Mantine Component.

**Reply by rtivital** [reply]
`withGlobalStyles` prop adds font-family styles to body tag, it will work with any component

**Reply by max-programming** [reply]
@rtivital I noticed it works with all components except for the `Title` component. 
I have to add it manually on the `Title` component for it to work

**Reply by rtivital** [reply]
https://mantine.dev/theming/typography/#change-fonts

**Reply by mislavperi** [reply]
AFAIK doesn't work for all fonts, Quicksand is an example of a font which it doesn't work with

**Comment by I-am-tk** [comment]
I am not able to style the `:disabled` state of `button` with `createStyle`  or with `sx`. Seems like the style set to the disabled state of the button is taking precedence over these rules.

See this https://codesandbox.io/s/modern-monad-uv3e4x?file=/src/App.tsx. 
I am having to use `!important` is there any better method. to style the `:disabled` state .

**Reply by rtivital** [reply]
Use important, it works fine, I do not see any issue here

**Comment by tianhuil** [comment]
FYI - I ended up using the example from https://github.com/orgs/mantinedev/discussions/1257#discussioncomment-6275498 and it worked.
